lms/djangoapps/courseware/access.py

```diff
@@ -270,7 +270,8 @@ def _can_enroll_courselike(user, courselike):
     if _has_staff_access_to_descriptor(user, courselike, course_key):
         return ACCESS_GRANTED
 
-    if courselike.invitation_only:
+    # Access denied when default value of COURSE_DEFAULT_INVITE_ONLY set to True
+    if settings.FEATURES.get('COURSE_DEFAULT_INVITE_ONLY') or courselike.invitation_only:
```

Comment:
So if the `COURSE_DEFAULT_INVITE_ONLY` flag is `True`, then it's not possible to override that behavior at the course level? If the site chooses to set that flag to `True`, and I set my course's `invitation_only` attribute to `False`, the site's setting takes precedence?

Reply:
The owner of the PR has better context  so @ormsbee I am jumping over this PR now, Can you please elaborate a bit more do you need any specific change here ?